proto/minder/v1/minder.proto

```diff
@@ -90,8 +90,14 @@ extend google.protobuf.MethodOptions {
 message Cursor {
     // cursor is the index to start from within the collection being
     // retrieved. It's an opaque payload specified and interpreted on
-    // an per-rpc basis.
-    string cursor = 1;
+    // an per-rpc basis. Note it is base64 encoded.
+    string cursor = 1 [
+        (buf.validate.field).string = {
+            pattern: "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$",
```

Comment:
Can we simplify this to `"^[[:word:]=]*$"`?

Reply:
The one you suggest seems more loose?

Reply:
It is.  We're concerned with very unexpected input, but presumably we will validate the cursor more completely after base64-decoding it, so we just want to rule out truly unexpected characters here.

Reply:
I see. Alright, make sense 👍